README.md

```diff
@@ -13,6 +13,30 @@ ImageKit is complete media storage, optimization, and transformation solution th
 
 ## Breaking changes
```

Comment:
Remove from here. Follow other frontend SDKs (Next, React, Vue) for changelog format.